code/go/0chain.net/blobbercore/handler/protocol.go

```diff
@@ -68,7 +68,6 @@ func getStorageNode() (*transaction.StorageNode, error) {
 	sn.Terms.WritePrice = zcncore.ConvertToValue(writePrice)
 	sn.Terms.MinLockDemand = config.Configuration.MinLockDemand
 	sn.Terms.MaxOfferDuration = config.Configuration.MaxOfferDuration
```

Comment:
Remove the ChallengeCompletionTime from config as well

Reply:
same as above comment

Reply:
fixed